hive-bigquery-connector-common/src/main/java/com/google/cloud/hive/bigquery/connector/utils/hive/HiveUtils.java

```diff
@@ -65,11 +65,9 @@ public static String getQueryId(Configuration conf) {
       // In this case, the user is running a plain Hive query directly from Hive itself.
       return "hive-query-id-" + hiveQueryId;
     }
-    if (conf.get("pig.script.id") != null) {
-      // The user is running a Hive query from Pig. Use the job's timestamp as a pig script might
-      // run multiple jobs.
-      return String.format(
-          "pig-%s-%s", conf.get("pig.script.id"), conf.get("pig.job.submitted.timestamp"));
+    if (conf.get("mapreduce.workflow.id") != null) {
```

Comment:
can we use fallback, first check pig.script.id if null check mapreduce.workflow.id?